languages/translations/de.json

```diff
@@ -84,23 +84,34 @@
         },
         "continue": {
             "default": {
-                "default": "{modifier} weiterfahren",
-                "name": "{modifier} weiterfahren auf {way_name}",
-                "destination": "{modifier} weiterfahren Richtung {destination}"
+                "default": "{modifier} abbiegen",
```

Comment:
Here and below: for all the `continue` maneuvers I think we should highlight that the road actually continues.